lighthouse-core/lib/i18n/i18n.js

```diff
@@ -6,19 +6,17 @@
 'use strict';
 
 const path = require('path');
-const isDeepEqual = require('lodash.isequal');
-const log = require('lighthouse-logger');
 const MessageFormat = require('intl-messageformat').default;
 const lookupClosestLocale = require('lookup-closest-locale');
 const LOCALES = require('./locales.js');
+const {isObjectOfUnknownValues, isObjectOrArrayOfUnknownValues} = require('../type-verifiers.js');
 
 /** @typedef {import('intl-messageformat-parser').Element} MessageElement */
 /** @typedef {import('intl-messageformat-parser').ArgumentElement} ArgumentElement */
 
 const LH_ROOT = path.join(__dirname, '../../../');
-const MESSAGE_INSTANCE_ID_REGEX = /(.* \| .*) # (\d+)$/;
```

Comment:
this thing is now completely eliminated, correct?

Reply:
> this thing is now completely eliminated, correct?

yes, now it's just a check for the locale json file keys (it could also be inlined in the single place it's used now)